cpp/src/gandiva/like_holder.h

```diff
@@ -48,6 +51,9 @@ class GANDIVA_EXPORT LikeHolder : public FunctionHolder {
  private:
   explicit LikeHolder(const std::string& pattern) : pattern_(pattern), regex_(pattern) {}
 
+  explicit LikeHolder(const std::string& pattern, RE2::Options regex_op)
```

Comment:
You don't need explicit here. explicit is used to prevent implicit conversions.

Reply:
done